swagger/api/v1/threads.yaml

```diff
@@ -0,0 +1,16 @@
+post:
+  operationId: createThread
+  summary: Creates a session/thread object to the server's persistent storage and returns the ID of the thread.
+  responses:
+    '201':
```

Comment:
Given it resturns the thread ID and thus has reserved the thread, it would make more sense to move to a 200 instead of a 201 right? There is nothing to be done on the server side afterwards except for when the next call comes in